### ISSUE TITLE
ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.GP17.AZAL], 

### ISSUE THREAD

**Issue by a7x-bit** [pop-os/pop#2769]
Recently noticed an "ACPI BIOS Error" during system boot. I am not sure if it was there since beginning, running on HP DevOne.

0.277424] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.GP17.AZAL], AE_NOT_FOUND (20220331/dswload2-162)
0.277430] ACPI Error: AE_NOT_FOUND, During name lookup/catalog (20220331/psobject-220)

**Distribution (run `cat /etc/os-release`):**
NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os

 
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

N/A

**Issue/Bug Description:**

0.277424] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.GP17.AZAL], AE_NOT_FOUND (20220331/dswload2-162)
0.277430] ACPI Error: AE_NOT_FOUND, During name lookup/catalog (20220331/psobject-220)

**Steps to reproduce (if you know):**

During boot time.

**Expected behavior:**

**Other Notes:**

System boots normal.

**Comment by leviport** [comment]
I don't see it during boot on my Dev One, but I do see it going way back in my journal logs. I'm not seeing any issues though, so it's probably safe to ignore the message.

As to why it's appearing all of a sudden, my only guess is that the `quiet`, `splash`, and/or `systemd.show_status=false` kernel parameters got removed. You can check if they're still there with `sudo kernelstub -p`, on the "Kernel Boot Options" line.

**Comment by a7x-bit** [comment]
Thanks,
looks like there are still there.

Kernel Boot Options:.quiet loglevel=0 systemd.show_status=false splash

I guess I'll just ignore for now.
Should I close the issue or keep it open?

**Comment by leviport** [comment]
I'll close it for now. It can be reopened if any negative behavior related to the error starts to show up.

**Comment by MichelN86** [comment]
Errors are still there. I have 

[    0.232567] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.XBAS], AE_NOT_FOUND (20220331/psargs-330)
[    0.232577] ACPI: Ignoring error and continuing table load
[    0.232633] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.OPG0], AE_NOT_FOUND (20220331/dsfield-500)
[    0.232655] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.XBAS], AE_NOT_FOUND (20220331/psargs-330)
[    0.232661] ACPI: Ignoring error and continuing table load
[    0.232673] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.SBN0], AE_NOT_FOUND (20220331/psargs-330)
[    0.232677] ACPI: Ignoring error and continuing table load
[    0.232697] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.PCS0], AE_NOT_FOUND (20220331/dsfield-500)
[    0.232715] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.XBAS], AE_NOT_FOUND (20220331/psargs-330)
[    0.232721] ACPI: Ignoring error and continuing table load
[    0.232733] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.SBN0], AE_NOT_FOUND (20220331/psargs-330)
[    0.232737] ACPI: Ignoring error and continuing table load
[    0.232751] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.EECP], AE_NOT_FOUND (20220331/psargs-330)
[    0.232755] ACPI: Ignoring error and continuing table load
[    0.232772] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.CAP0], AE_NOT_FOUND (20220331/dsfield-500)
[    0.232790] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.XBAS], AE_NOT_FOUND (20220331/psargs-330)
[    0.232796] ACPI: Ignoring error and continuing table load
[    0.232840] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.OPG1], AE_NOT_FOUND (20220331/dsfield-500)
[    0.232859] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.XBAS], AE_NOT_FOUND (20220331/psargs-330)
[    0.232865] ACPI: Ignoring error and continuing table load
[    0.232877] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.SBN1], AE_NOT_FOUND (20220331/psargs-330)
[    0.232881] ACPI: Ignoring error and continuing table load
[    0.232902] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.PCS1], AE_NOT_FOUND (20220331/dsfield-500)
[    0.232920] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.XBAS], AE_NOT_FOUND (20220331/psargs-330)
[    0.232926] ACPI: Ignoring error and continuing table load
[    0.232939] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.SBN1], AE_NOT_FOUND (20220331/psargs-330)
[    0.232943] ACPI: Ignoring error and continuing table load
[    0.232957] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.EEC1], AE_NOT_FOUND (20220331/psargs-330)
[    0.232961] ACPI: Ignoring error and continuing table load
[    0.232978] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.CAP1], AE_NOT_FOUND (20220331/dsfield-500)
[    0.232996] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.XBAS], AE_NOT_FOUND (20220331/psargs-330)
[    0.233002] ACPI: Ignoring error and continuing table load
[    0.233047] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.OPG2], AE_NOT_FOUND (20220331/dsfield-500)
[    0.233067] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.XBAS], AE_NOT_FOUND (20220331/psargs-330)
[    0.233074] ACPI: Ignoring error and continuing table load
[    0.233086] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.SBN2], AE_NOT_FOUND (20220331/psargs-330)
[    0.233090] ACPI: Ignoring error and continuing table load
[    0.233111] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.PCS2], AE_NOT_FOUND (20220331/dsfield-500)
[    0.233130] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.XBAS], AE_NOT_FOUND (20220331/psargs-330)
[    0.233137] ACPI: Ignoring error and continuing table load
[    0.233149] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.SBN2], AE_NOT_FOUND (20220331/psargs-330)
[    0.233153] ACPI: Ignoring error and continuing table load
[    0.233168] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.EEC2], AE_NOT_FOUND (20220331/psargs-330)
[    0.233172] ACPI: Ignoring error and continuing table load
[    0.233190] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.CAP2], AE_NOT_FOUND (20220331/dsfield-500)
[    0.233208] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.XBAS], AE_NOT_FOUND (20220331/psargs-330)
[    0.233214] ACPI: Ignoring error and continuing table load
[    0.233230] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.MCD0], AE_NOT_FOUND (20220331/dsfield-500)

and

[    0.320655] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.XHC.RHUB.TPLD], AE_NOT_FOUND (20220331/psargs-330)

[    0.320664] No Local Variables are initialized for Method [_PLD]

[    0.320667] No Arguments are initialized for method [_PLD]

[    0.320669] ACPI Error: Aborting method \_SB.UBTC.CR01._PLD due to previous error (AE_NOT_FOUND) (20220331/psparse-529)
[    0.322443] ACPI: PCI Root Bridge [PCI0] (domain 0000 [bus 00-fe])

**Comment by MichelN86** [comment]
Running Pop! OS 22.04 kernel 6.0.12-76060006-generic

**Comment by a7x-bit** [comment]
> I'll close it for now. It can be reopened if any negative behavior related to the error starts to show up.

Roger,
Thank you.

**Comment by thimslugga** [comment]
I started seeing the same thing during boot with Fedora 38 Silverblue on a HP Dev One

**Comment by alexdelorenzo** [comment]
Got this on Arch with Linux 6.4.x on the HP Dev One:

    [    0.473582] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.GP17.AZAL], AE_NOT_FOUND (20230331/dswload2-162)
    [    0.473605] ACPI Error: AE_NOT_FOUND, During name lookup/catalog (20230331/psobject-220)

**Comment by MartinVonReichenberg** [comment]
Reporting it too,
The similar **BUG** occurs in **_any_** Linux distribution:

```
martin@Kingston-USB-Flash-Drive--Ubuntu-23-10 ~> sudo dmesg | sort | grep -ie error
[sudo] password for martin: 
[    0.286956] ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.PB2], AE_NOT_FOUND (20230331/dswload2-162)
[    0.286977] ACPI Error: AE_NOT_FOUND, During name lookup/catalog (20230331/psobject-220)
[    0.725659] RAS: Correctable Errors collector initialized.
[   16.770579] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  559.025436] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  559.025514] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  559.025555] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  559.025593] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  619.309724] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  684.161067] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  684.161175] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  684.161217] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  684.161256] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.582457] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.582538] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.582579] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.582616] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.666306] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.666387] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.666425] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.666461] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.753582] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.753663] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.753702] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
[  688.753738] [drm:nv_drm_master_set [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to grab modeset ownership
```

```
martin@Kingston-USB-Flash-Drive--Ubuntu-23-10 ~> sudo inxi -F
System:
  Host: Kingston-USB-Flash-Drive--Ubuntu-23-10 Kernel: 6.5.0-9-generic
    arch: x86_64 bits: 64 Desktop: GNOME v: 45.0 Distro: Ubuntu 23.10 (Mantic
    Minotaur)
Machine:
  Type: Laptop System: LENOVO product: 82JQ v: Legion 5 Pro 16ACH6H
    serial: PF43CAE5
  Mobo: LENOVO model: LNVNB161216 v: NO DPK serial: PF43CAE5 UEFI: LENOVO
    v: GKCN60WW date: 03/07/2023
Battery:
  ID-1: BAT0 charge: 78.7 Wh (94.0%) condition: 83.7/80.0 Wh (104.6%)
CPU:
  Info: 6-core model: AMD Ryzen 5 5600H with Radeon Graphics bits: 64
    type: MT MCP cache: L2: 3 MiB
  Speed (MHz): avg: 626 min/max: 400/4280 cores: 1: 2119 2: 400 3: 400
    4: 400 5: 400 6: 400 7: 400 8: 1397 9: 400 10: 400 11: 400 12: 400
Graphics:
  Device-1: NVIDIA GA106M [GeForce RTX 3060 Mobile / Max-Q] driver: nvidia
    v: 535.113.01
  Display: server: X.Org v: 1.21.1.7 with: Xwayland v: 23.2.0 driver: X:
    loaded: nvidia unloaded: fbdev,modesetting,nouveau,vesa
    gpu: nvidia,nvidia-nvswitch resolution: 3408x2130
  API: OpenGL v: 4.6.0 NVIDIA 535.113.01 renderer: NVIDIA GeForce RTX 3060
    Laptop GPU/PCIe/SSE2
Audio:
  Device-1: NVIDIA GA106 High Definition Audio driver: snd_hda_intel
  Device-2: AMD ACP/ACP3X/ACP6x Audio Coprocessor driver: N/A
  Device-3: AMD Family 17h/19h HD Audio driver: snd_hda_intel
  API: ALSA v: k6.5.0-9-generic status: kernel-api
Network:
  Device-1: Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet
    driver: r8169
  IF: eno1 state: up speed: 1000 Mbps duplex: full mac: 9c:2d:cd:c9:f6:9b
  Device-2: MEDIATEK MT7921 802.11ax PCI Express Wireless Network Adapter
    driver: mt7921e
  IF: wlp4s0 state: down mac: f0:a6:54:33:f4:2d
  Device-3: Realtek RTL8153 Gigabit Ethernet Adapter driver: r8152 type: USB
  IF: enx9405bb1970c2 state: down mac: 94:05:bb:19:70:c2
Bluetooth:
  Device-1: Foxconn / Hon Hai MediaTek Bluetooth Adapter driver: btusb
    type: USB
  Report: hciconfig ID: hci0 rfk-id: 2 state: down
    bt-service: enabled,running rfk-block: hardware: no software: yes
    address: F0:A6:54:33:F4:2E
Drives:
  Local Storage: total: 505.77 GiB used: 22.8 GiB (4.5%)
  ID-1: /dev/nvme0n1 vendor: Samsung model: MZVLB512HBJQ-000L2
    size: 476.94 GiB
  ID-2: /dev/sda vendor: Kingston model: DataTraveler 3.0 size: 28.83 GiB
    type: USB
Partition:
  ID-1: / size: 27.17 GiB used: 22.79 GiB (83.9%) fs: ext4 dev: /dev/sda2
  ID-2: /boot/efi size: 1.05 GiB used: 6.1 MiB (0.6%) fs: vfat
    dev: /dev/sda1
Swap:
  ID-1: swap-1 type: file size: 4 GiB used: 0 KiB (0.0%) file: /swap.img
Sensors:
  System Temperatures: cpu: 38.8 C mobo: N/A gpu: nvidia temp: 40 C
  Fan Speeds (rpm): N/A
Info:
  Processes: 336 Uptime: 18m Memory: total: 16 GiB available: 15.46 GiB
  used: 2.8 GiB (18.1%) Shell: Sudo inxi: 3.3.29
```
